src/Barcoder/DataMatrix/DataMatrixEncoder.cs

```diff
@@ -6,12 +6,23 @@ namespace Barcoder.DataMatrix
 {
     public static class DataMatrixEncoder
     {
-        public static IBarcode Encode(string content)
+        public static IBarcode Encode(string content, byte codeSize = 0)
```

Comment:
`codeSize` seems to be the wrong name for this variable. I think I'd need to know the use-case in order to understand why this change is needed. I guess that you want to have a fixed size matrix for whatever content size, is that correct?

Reply:
After understanding why this is needed, perhaps `int? fixedNumberOfRows = null` would be better as parameter.